dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpChannelConfig.java

```diff
@@ -2522,14 +2496,12 @@ private long convertLong(Object o) {
      */
     private int rangeLimit(int size, int min, int max) {
         if (size < min) {
-            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-                Tr.debug(tc, "Config: " + size + " too small");
-            }
+            Tr.debug(tc, "Config: " + size + " too small");
```

Comment:
Should if be contained within "if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {" ? 

Second time I see this.